com.unity.render-pipelines.universal/Editor/DeferredRendererDataEditor.cs

```diff
@@ -19,6 +19,7 @@ private static class Styles
             public static readonly GUIContent preferDepthPrepassLabel = EditorGUIUtility.TrTextContent("Prefer Depth Prepass", "Some platform configurations may still enable depth-prepass even when this checkbox is disabled.");
             public static readonly GUIContent accurateGbufferNormalsLabel = EditorGUIUtility.TrTextContent("Accurate G-buffer normals", "normals in G-buffer use octaedron encoding/decoding (expensive)");
             public static readonly GUIContent tiledDeferredShadingLabel = EditorGUIUtility.TrTextContent("Tiled Deferred Shading (Experimental)", "Allows Tiled Deferred Shading on appropriate lights");
+            public static readonly GUIContent invalidStencilOverride = EditorGUIUtility.TrTextContent("Deferred renderer reserves the 4 highest bits of stencil to store material types. Stencil Overrides settings are constrained to ensure correct lighting results.\n");
```

Comment:
@kaychang-unity , I need help in understanding this text. I'll ping you on Slack.